components/nimbus/tests/test_telemetry.rs

```diff
@@ -0,0 +1,219 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+mod common;
+#[cfg(feature = "rkv-safe-mode")]
+#[cfg(test)]
+mod test {
+
+    use super::common::new_test_client_with_db;
+    use glean::{test_reset_glean, ClientInfoMetrics, Configuration, RecordedEvent};
+    use nimbus::glean_metrics;
+
+    #[cfg(feature = "rkv-safe-mode")]
```

Comment:
nit: do we need this? would the `cfg` on the `test` module protect against this already?

Reply:
I think it might, this is most likely just some copy-pasta from Glean test examples.